app/views/hyrax/contact_form/new.html.erb

```diff
@@ -18,35 +18,36 @@
 <% end %>
 
 <%= form_for @contact_form, url: hyrax.contact_form_index_path,
-                            html: { class: 'form-horizontal' } do |f| %>
+                            html: { class: '' } do |f| %>
   <%= raw negative_captcha(@captcha) %>
   <%= f.text_field :contact_method, class: 'hide' %>
-  <div class="form-group">
-    <%= f.label :category, t('hyrax.contact_form.type_label'), class: "col-sm-2 control-label" %>
+
+  <div class="form-group row">
+    <%= f.label :category, t('hyrax.contact_form.type_label'), class: "col-sm-2 col-form-label" %>
     <div class="col-sm-10">
       <%= f.select 'category', options_for_select(contact_form_issue_type_options), { include_blank: t('hyrax.contact_form.select_type') }, {class: 'form-control', required: true } %>
     </div>
   </div>
 
-  <div class="form-group">
-    <%= negative_label_tag(@captcha, :name, t('hyrax.contact_form.name_label'), class: "col-sm-2 control-label" ) %>
```

Comment:
🟡 Curious about what the `negative_label_tag` and `negative_text_field_tag` are all about.  This looks to have been introduced via `Hyrax::ContactFormController#setup_negative_captcha`

Without this, do we still have Captcha?  The contact form is definitely something that get's spammed as well as attacked as a test vector for security issues.

Reply:
Good catch, this is an oversight